src/Shift-ClassBuilder.package/ClassOrganization.extension/instance/copyFrom..st

```diff
@@ -0,0 +1,9 @@
+*Shift-ClassBuilder
+copyFrom: otherOrganization
+	comment := otherOrganization instVarNamed: #comment.
```

Comment:
I think it should be converted to copy-postCopy to avoid direct instance variables access